src/Tribe/Tickets.php

```diff
@@ -3460,10 +3460,10 @@ public function maybe_show_tickets_unavailable_message( $tickets, $post_id = nul
 				 *
 				 * @since TBD
 				 *
-				 * @param string Unavailability message
-				 * @param array Collection of tickets
+				 * @param $message string Unavailability message
+				 * @param $tickets array Collection of tickets
 				 */
-				echo (string) apply_filters( 'event_tickets_unvailable_message', $message, $tickets );
+				echo (string) apply_filters( 'tec_tickets_unavailable_message', $message, $tickets );
```

Comment:
This is an existing filter and we should not rename it. Instead let's deprecate the previous one and then add the new filter as well.